be/src/common/config.h

```diff
@@ -756,6 +756,7 @@ CONF_mBool(orc_coalesce_read_enable, "true");
 // parquet reader
 CONF_mBool(parquet_coalesce_read_enable, "true");
 CONF_Bool(parquet_late_materialization_enable, "true");
```

Comment:
false here?

Reply:
I think in main branch "true" is ok, If we think it's not stable, we till turn off it in release version.